onnxruntime/python/tools/kernel_explorer/kernels/gemm_fast_gelu_test.py

```diff
@@ -0,0 +1,197 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import re
+import sys
+from itertools import product
+
+import kernel_explorer as ke
+import numpy as np
+import pytest
+from utils import get_gemm_bound
+
+
+def transab_to_suffix(transab):
+    return {
+        (True, True): "TT",
+        (True, False): "TN",
+        (False, True): "NT",
+        (False, False): "NN",
+    }[tuple(transab)]
+
+
+def dtype_to_funcs(dtype):
+    type_map = {
+        "float16": list(filter(lambda x: re.search("GemmFastGelu.*_half", x), dir(ke))),
+        "float32": list(filter(lambda x: re.search("GemmFastGelu.*_float", x), dir(ke))),
+    }
+    return type_map[dtype]
+
+
+def fast_gelu(x, bias):
+    x = x + bias
+    y = 0.5 * x * (1 + np.tanh(0.797885 * x + 0.035677 * x * x * x))
+    return y
+
+
+# TODO The test method needs update.
```

Comment:
What update is needed?

Reply:
Because the bound only calculate for GEMM op and not consider FastGelu, for some case the bound is too strict. But I not found a better way to calculate the bound yet.

Reply:
Please add the reply to code comment.